frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -1184,7 +1184,7 @@
                 },
                 onError: function(response) {
                     var error = response.data;
-                    $rootScope.notify("error", error);
+                    $rootScope.notify("error", "Login Required !");
```

Comment:
Please redirect the user to login page.

The flow should be - 
1. User clicks on the `star challenge` button.
2. The user is redirected to a login page.
3. A user logs in. After logging in, the user is redirected to the challenge overview tab.

Reply:
@RishabhJain2018 Done the changes, Please review.